docs/api/samples/concept-in-scheme.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+      "dc": "http://purl.org/dc/elements/1.1/",
+      "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+      "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+      "skos": "http://www.w3.org/2004/02/skos/core#",
+      "xsd": "http://www.w3.org/2001/XMLSchema#"
+    },
+    "@id": "urn:wbs:47",
+    "@type": "skos:Concept",
+    "skos:broader": [{
+      "@id": "urn:wbs:42"
+    }],
+    "skos:inScheme": [{
+      "@id": "myscheme"
+    }],
+    "skos:prefLabel": "Trajectory analysis"
```

Comment:
Are the `skos:prefLabel` going to come back normalised as a language map (with the default language)?

Reply:
Yes, normalised as a language map. You'll need to be able to determine the current language somehow (JWT?). I should update this example to reflect that.

Reply:
MVP will just be configured at build time in the FE, but yeh, will be up to the FE to display the right one and fallback as needed.